report_py3o/models/py3o_template.py

```diff
@@ -22,3 +23,12 @@ class Py3oTemplate(models.Model):
         required=True,
         default="odt",
     )
+
+
+class Py3oTemplateMixin(models.AbstractModel):
```

Comment:
Better use a different file and make sure you load it before the others in the `__init__` to avoid issues

Reply:
A separate file causes a recurrence error
```
ImportError: cannot import name 'py3o_template_mixin' from partially initialized module 'odoo.addons.report_py3o.models' (most likely due to a circular import) (/opt/odoo/auto/addons/report_py3o/models/__init__.py)
```